Makefile

```diff
@@ -24,17 +24,25 @@ deps:
 checks:
 	./scripts/checks "$(PKGS)"
 
-build: deps checks
+run-build: deps checks
 	godep go install -v $(TO_BUILD)
 
+build: start
```

Comment:
I think we can add a direct dependency for `deps` on the `build` target.

This will remove the need for downloading godep and golint inside the vagrant vm. See next comment on keeping the `CONTIV_HOST_GOPATH` env